js/binance.js

```diff
@@ -3660,7 +3660,7 @@ module.exports = class binance extends Exchange {
                 tag = undefined;
             }
         }
-        let txid = this.safeString (transaction, 'txId');
+        let txid = this.safeString (transaction, 'txId', '');
```

Comment:
I don't think that we should fill in these strings, if they're missing. They should not be empty.

Reply:
@samgermain can you revert the commit that added the edits in derived files in this PR?

Reply:
@kroitor All the ones that get a default of an empty string get it because we later call something like `.length` or `.indexOf` on it, so the program will crash if it's set to undefined,